src/Page.js

```diff
@@ -669,6 +682,10 @@ class Page {
 
   collectHeadFiles(baseUrl, hostBaseUrl) {
     const { head } = this.frontMatter;
+    if (head === FRONT_MATTER_NONE_ATTR) {
```

Comment:
I believe you have to set

```
this.headFileTopContent = '';
this.headFileBottomContent = '';
```

If not this, there could be an issue when changing the file on live reload in this sequence:

1. Add layout
2. Add `head: none`

But the `headFileTopContent` is not removed.

Reply:
Yes, you're right. Will make the change 👍